acc-testcases/data-sources/load_balancer.yaml

```diff
@@ -0,0 +1,3 @@
+acc:
+- config: |
+    name = "tf_loadbalancer"
```

Comment:
I hope you have cleaned up everything except this resource. Can you add a validation here to check the ID and any other fields if any.

Reply:
yes